.github/codecov.yml

```diff
@@ -0,0 +1,21 @@
+codecov:
+  max_report_age: off
+
+comment: false
+
+coverage:
+  precision: 2
+  round: down
+  status:
+    project:
+      default:
+        threshold: 0.2
+        if_not_found: success
+    patch:
+      default:
+        enabled: no
+        if_not_found: success
+    changes:
+      default:
+        enabled: no
+        if_not_found: success
```

Comment:
This file shouldn't be here. It should be in the root directory.

Reply:
> This file shouldn't be here. It should be in the root directory.

I don't think that's an issue since (1) [codecov docs](https://docs.codecov.io/docs/codecov-yaml) do not explicitly say it should be in the root directory, (2) GeoCAT-comp also has that file under `.github` folder and its badge works fine:

![image](https://user-images.githubusercontent.com/32553057/112698013-60597200-8e4e-11eb-9635-0b6042ac0a1b.png)


But, when you check README of this branch here (`update_actions`), it cannot show the report outputs:

![image](https://user-images.githubusercontent.com/32553057/112698105-939c0100-8e4e-11eb-8123-d395a4a9e8a3.png)

I am checking the codecov badge and URL, and they seem like good, too.

Reply:
Ok, I now realize there is nothing to be concerned here since the same warning (i.e. "repo is not enabled") is also being given for GeoCAT-comp but its overview is correctly populated from `main` branch. Once this PR is merged, the badge here as well as the overview at codecov should be correctly populated. 

Thanks for discussing!